template-manager-cmtemplate/src/main/java/com/sequenceiq/cloudbreak/cmtemplate/configproviders/hbase/HbaseRestKnoxServiceConfigProvider.java

```diff
@@ -29,17 +25,9 @@ public class HbaseRestKnoxServiceConfigProvider implements CmTemplateComponentCo
     @Inject
     private ExposedServiceCollector exposedServiceCollector;
 
-    @Inject
-    private EntitlementService entitlementService;
-
     @Override
     public List<ApiClusterTemplateConfig> getServiceConfigs(CmTemplateProcessor templateProcessor, TemplatePreparationObject templatePreparationObject) {
-        List<ApiClusterTemplateConfig> configs = new ArrayList<>();
-        configs.add(config(RESTSERVER_SECURITY_AUTHENTICATION, "kerberos"));
-        if (isSDXOptimizationNeeded(templatePreparationObject)) {
-            configs.add(config(HBASE_RPC_PROTECTION, "privacy"));
-        }
-        return configs;
+        return List.of(config(RESTSERVER_SECURITY_AUTHENTICATION, "kerberos"), config(HBASE_RPC_PROTECTION, "privacy"));
```

Comment:
This change should still be under an entitlement. we know there are integration DH issues with this rpc protection configuration.